reference/admin.yaml

```diff
@@ -8185,6 +8185,217 @@ components:
         - matched_expenses
         - last_user_matched_at
         - metadata
+    amex_account_in:
+      type: object
+      additionalProperties: false
+      properties:
+        id:
+          $ref: '#/components/schemas/id_string'
+          example: amacwbl222wlg
+          nullable: true
+        created_at:
+          $ref: '#/components/schemas/created_at'
+          nullable: true
+        updated_at:
+          $ref: '#/components/schemas/updated_at'
+          nullable: true
+        user_id:
```

Comment:
why we are using user_id and org_id in the request body... This should always be calculated from the auth token .